src/map/clif.cpp

```diff
@@ -16698,7 +16700,11 @@ void clif_parse_ViewPlayerEquip(int fd, struct map_session_data* sd)
 	if (!tsd)
 		return;
 
-	if( tsd->status.show_equip || pc_has_permission(sd, PC_PERM_VIEW_EQUIPMENT) )
+	if (pc_has_permission(sd, PC_PERM_VIEW_EQUIPMENT))
```

Comment:
Should still be on the same map or not?

Reply:
because it's admin permission, that's why ignore the map restriction.
or what? this from packet request always strict the map, then make atcommand that ignore map and map-server restrtiction /gg

Reply:
I dont think adding „features“ like this intentionally is a good idea.
If we forgot to add a check somewhere and you know of it feel free to add the missing ones though or fix the order.

Reply:
:thinking: what is this, just like I'm sweeping dirty floor and someone comes ask for "are you clearing this properly" while checking the dirts and make a mess, then said "oh, seems nothing there what I'd like to find"
:rofl:

Reply:
You are not clearing it properly. You clear one dirt pile and create a dirt pile closeby. 
So telling you this is like:
You did a good start, but you might be able to improve it even further. It is no offense.